docs/cli.rst

```diff
@@ -234,12 +234,13 @@ or do a round trip like this
     [106, 193, 9]
 
 If you have `geojsonio-cli <https://github.com/mapbox/geojsonio-cli>`__
-installed, you can shoot this GeoJSON straight to `geojson.io
+and `fiona <https://pypi.python.org/pypi/Fiona>`_ installed, you can shoot this GeoJSON straight to `geojson.io
 <http://geojson.io/>`__ for lightning-fast visualization and editing.
 
 .. code-block:: console
 
     $ echo "[-105, 39.99, -104.99, 40]" \
-    > | mercantile tiles 14 \
-    > | mercantile shapes --compact \
-    > | geojsonio
+    | mercantile tiles 14 \
+    | mercantile shapes \
```

Comment:
@samsammurphy instead of adding fiona in the mix (which will work) let's put `mercantile shapes --collect` (writes shapes as a feature collection) here. I think it's possible that was my original intent and that `--compact` was a slip up.

Reply:
what ?  there is a `--collect` option 🤦‍♂️ 

thanks @sgillies